### PR TITLE
Tmedia 838 image arrows counter

### PR DESCRIPTION
**Pull request by JackHowa**
## Ticket

- [TMEDIA-838](https://arcpublishing.atlassian.net/browse/TMEDIA-838) sub-task of https://arcpublishing.atlassian.net/jira/software/c/projects/TMEDIA/boards/875?modal=detail&selectedIssue=TMEDIA-689&assignee=5e387d6a1b1d910e5dfd7a9b

## Description

Additional image arrows that can be toggled to show and hide via theme file 

## Acceptance Criteria

Image arrows near the image counter drop off at the mobile breakpoint (the large arrows on the image itself stay)

## Test Steps

1. Checkout branch - `git checkout TMEDIA-838-image-arrows-counter`
2. Run Storybook `npm run storybook`
3. See the additional default buttons story. Make sure you're on commerce theme
Screen Shot 2022-04-19 at 13 43 54
4. Click next ">" arrow and see the other previous "<" show. This can be spaced by the theme file 
Screen Shot 2022-04-19 at 13 44 55
5. Go to the custom additional controls story. 
6. See the various updates to the inline custom buttons
Screen Shot 2022-04-19 at 13 46 41

Screen Shot 2022-04-19 at 13 46 54
7. In the commerce theme, shrink the screen to mobile. See the additional arrows no longer visible via display: none

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed relevant documentation has been updated/added.
- [x] Add label - **ready for review** when the pull request is ready for someone to begin reviewing

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] All GitHub Actions pass
- [ ] Ran the code locally based on the test instructions.
- [ ] Checked Chromatic for Storybook changes, accepted the updates if acceptable
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.
- [ ] Approve and Add label - **ready to merge** if you are happy with the pull request
- [ ] Want another reviewer? Add the label **additional review**

### CODE REVIEW

**Comment by matthewroach** [comment]
@JackHowa - Same question/comment as on the other PR - https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266

**Comment by matthewroach** [comment]
@JackHowa - Same question/comment as on the other PR - https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266

**Comment by JackHowa** [comment]
@matthewroach updated

> @JackHowa - Same question/comment as on the other PR - [#73 (comment)](https://github.com/WPMedia/arc-themes-components/pull/73#issuecomment-1103794266)

Screen Shot 2022-04-20 at 09 17 19

**Review by nschubach — APPROVED** [review]
Looks great to me, covers the requirement.

**Comment by JackHowa** [comment]
@matthewroach updated to align items within the controls container next to each other horizontally rather than vertically

File: src/components/carousel/index.jsx [hunk 1]
```diff
@@ -29,6 +29,28 @@ const DefaultPreviousButton = ({ id, onClick }) => (
...
 );
 
+const DefaultAdditionalPreviousButton = ({ id, onClick }) => (
+	<Button
+		id={id}
+		onClick={onClick}
+		label="Previous Slide"
+		className={`${COMPONENT_CLASS_NAME}__button ${COMPONENT_CLASS_NAME}__button--additional-previous`}
+	>
+		{"<"}
```
Comment (matthewroach) [inline src/components/carousel/index.jsx]: Can use use an `<Icon />` here? `ChevronLeft`

File: src/components/carousel/index.jsx [hunk 2]
```diff
@@ -29,6 +29,28 @@ const DefaultPreviousButton = ({ id, onClick }) => (
...
+);
+
+const DefaultAdditionalNextButton = ({ id, onClick }) => (
+	<Button
+		id={id}
+		onClick={onClick}
+		label="Next Slide"
+		className={`${COMPONENT_CLASS_NAME}__button ${COMPONENT_CLASS_NAME}__button--additional-next`}
+	>
+		{">"}
```
Comment (matthewroach) [inline src/components/carousel/index.jsx]: Can use use an `<Icon />` here? `ChevronLeft`

File: src/components/carousel/index.jsx (hunk 1 above)
Comment (JackHowa) [reply src/components/carousel/index.jsx]: sure. importing one component into the other ok?

Comment (matthewroach) [reply src/components/carousel/index.jsx]: Good question - we've not doing it yet - maybe we use an HTML entity then rather than the text >

Comment (JackHowa) [reply src/components/carousel/index.jsx]: @matthewroach updated
Screen Shot 2022-04-20 at 16 20 04

**Comment by JackHowa** [comment]
fixing test for ><

File: src/components/carousel/index.jsx (hunk 1 above)
Comment (JackHowa) [reply src/components/carousel/index.jsx]: @matthewroach updated 
Screen Shot 2022-04-20 at 16 20 04

**Review by matthewroach — APPROVED** [review]

**Pull request merged**